packages/rest/test/integration/rest-server.integration.ts

```diff
@@ -37,77 +37,12 @@ describe('RestServer (integration)', () => {
       .expect(500);
   });
 
-  it('exposes "GET /swagger.json" endpoint', async () => {
-    const server = await givenAServer({rest: {port: 0}});
-    const greetSpec = {
-      responses: {
-        200: {
-          schema: {type: 'string'},
-          description: 'greeting of the day',
-        },
-      },
-    };
-    server.route(new Route('get', '/greet', greetSpec, function greet() {}));
-
-    const response = await createClientForHandler(server.handleHttp).get(
-      '/swagger.json',
-    );
-    expect(response.body).to.containDeep({
-      basePath: '/',
-      paths: {
-        '/greet': {
-          get: greetSpec,
-        },
-      },
-    });
-    expect(response.get('Access-Control-Allow-Origin')).to.equal('*');
-    expect(response.get('Access-Control-Allow-Credentials')).to.equal('true');
-    expect(response.get('Access-Control-Allow-Max-Age')).to.equal('86400');
-  });
-
-  it('exposes "GET /swagger.yaml" endpoint', async () => {
-    const server = await givenAServer({rest: {port: 0}});
-    const greetSpec = {
-      responses: {
-        200: {
-          schema: {type: 'string'},
-          description: 'greeting of the day',
-        },
-      },
-    };
-    server.route(new Route('get', '/greet', greetSpec, function greet() {}));
-
-    const response = await createClientForHandler(server.handleHttp).get(
-      '/swagger.yaml',
-    );
-    const expected = yaml.safeLoad(`
-swagger: '2.0'
-basePath: /
-info:
-  title: LoopBack Application
-  version: 1.0.0
-paths:
-  /greet:
-    get:
-      responses:
-        '200':
-          schema:
-            type: string
-          description: greeting of the day
-    `);
-    // Use json for comparison to tolerate textual diffs
-    expect(yaml.safeLoad(response.text)).to.eql(expected);
-    expect(response.get('Access-Control-Allow-Origin')).to.equal('*');
-    expect(response.get('Access-Control-Allow-Credentials')).to.equal('true');
-    expect(response.get('Access-Control-Allow-Max-Age')).to.equal('86400');
-  });
-
   it('exposes "GET /openapi.json" endpoint', async () => {
     const server = await givenAServer({rest: {port: 0}});
     const greetSpec = {
       responses: {
         200: {
-          schema: {type: 'string'},
+          content: {'*/*': {schema: {type: 'string'}}},
```

Comment:
Similarly here, the endpoint should specify `text/plain` as the response content type.